flask_celeryext/_mapping.py

```diff
@@ -2,114 +2,128 @@
+    'CELERYBEAT_MAX_LOOP_INTERVAL': 'beat_max_loop_interval',
+    'CELERYBEAT_SCHEDULE': 'beat_schedule',
+    'CELERYBEAT_SCHEDULER': 'beat_scheduler',
+    'CELERYBEAT_SCHEDULE_FILENAME': 'beat_schedule_filename',
+    'CELERYBEAT_SYNC_EVERY': 'beat_sync_every',
+    'BROKER_URL': 'broker_url',
+    'BROKER_TRANSPORT': 'broker_transport',
+    'BROKER_TRANSPORT_OPTIONS': 'broker_transport_options',
+    'BROKER_CONNECTION_TIMEOUT': 'broker_connection_timeout',
+    'BROKER_CONNECTION_RETRY': 'broker_connection_retry',
+    'BROKER_CONNECTION_MAX_RETRIES': 'broker_connection_max_retries',
+    'BROKER_FAILOVER_STRATEGY': 'broker_failover_strategy',
+    'BROKER_HEARTBEAT': 'broker_heartbeat',
+    'BROKER_LOGIN_METHOD': 'broker_login_method',
+    'BROKER_POOL_LIMIT': 'broker_pool_limit',
+    'BROKER_USE_SSL': 'broker_use_ssl',
+    'CELERY_CACHE_BACKEND': 'cache_backend',
+    'CELERY_CACHE_BACKEND_OPTIONS': 'cache_backend_options',
+    'CASSANDRA_COLUMN_FAMILY': 'cassandra_table',
+    'CASSANDRA_ENTRY_TTL': 'cassandra_entry_ttl',
+    'CASSANDRA_KEYSPACE': 'cassandra_keyspace',
+    'CASSANDRA_PORT': 'cassandra_port',
+    'CASSANDRA_READ_CONSISTENCY': 'cassandra_read_consistency',
+    'CASSANDRA_SERVERS': 'cassandra_servers',
+    'CASSANDRA_WRITE_CONSISTENCY': 'cassandra_write_consistency',
+    'CASSANDRA_OPTIONS': 'cassandra_options',
+    'S3_ACCESS_KEY_ID': 's3_access_key_id',
+    'S3_SECRET_ACCESS_KEY': 's3_secret_access_key',
+    'S3_BUCKET': 's3_bucket',
+    'S3_BASE_PATH': 's3_base_path',
+    'S3_ENDPOINT_URL': 's3_endpoint_url',
+    'S3_REGION': 's3_region',
+    'CELERY_COUCHBASE_BACKEND_SETTINGS': 'couchbase_backend_settings',
+    'CELERY_ARANGODB_BACKEND_SETTINGS': 'arangodb_backend_settings',
+    'CELERY_MONGODB_BACKEND_SETTINGS': 'mongodb_backend_settings',
+    'CELERY_EVENT_QUEUE_EXPIRES': 'event_queue_expires',
+    'CELERY_EVENT_QUEUE_TTL': 'event_queue_ttl',
+    'CELERY_EVENT_QUEUE_PREFIX': 'event_queue_prefix',
+    'CELERY_EVENT_SERIALIZER': 'event_serializer',
+    'CELERY_REDIS_DB': 'redis_db',
+    'CELERY_REDIS_HOST': 'redis_host',
+    'CELERY_REDIS_MAX_CONNECTIONS': 'redis_max_connections',
+    'CELERY_REDIS_USERNAME': 'redis_username',
+    'CELERY_REDIS_PASSWORD': 'redis_password',
+    'CELERY_REDIS_PORT': 'redis_port',
+    'CELERY_REDIS_BACKEND_USE_SSL': 'redis_backend_use_ssl',
+    'CELERY_RESULT_BACKEND': 'result_backend',
+    'CELERY_MAX_CACHED_RESULTS': 'result_cache_max',
+    'CELERY_MESSAGE_COMPRESSION': 'result_compression',
+    'CELERY_RESULT_EXCHANGE': 'result_exchange',
+    'CELERY_RESULT_EXCHANGE_TYPE': 'result_exchange_type',
+    'CELERY_RESULT_EXPIRES': 'result_expires',
+    'CELERY_RESULT_PERSISTENT': 'result_persistent',
+    'CELERY_RESULT_SERIALIZER': 'result_serializer',
+    'CELERY_RESULT_DBURI': 'Use result_backend instead.',
+    'CELERY_RESULT_ENGINE_OPTIONS': 'database_engine_options',
+    '[...]_DB_SHORT_LIVED_SESSIONS': 'database_short_lived_sessions',
```

Comment:
This looks like a type (``[...]``)?

Reply:
i copy-pasted the mapping from https://docs.celeryproject.org/en/stable/userguide/configuration.html#new-lowercase-settings. i was not sure about that line too, but i thought if they have it in there it should be there, but after some thoughts it make no sense, because a variable name could not look like that. 

i did some research and the original variable is CELERY_RESULT_DB_SHORT_LIVED_SESSIONS. i will change that entry to that variable name